tools/query_breakdown/src/main/java/com/google/bigquery/ReplacedComponent.java

```diff
@@ -0,0 +1,46 @@
+package com.google.bigquery;
+
+/**
+ * This class captures the replaced query, original component, and what it is replaced with
+ */
+public class ReplacedComponent {
+  private String query;
```

Comment:
final modifier??

Reply:
Made all the fields final since replacedcomponent object will never change it's parameters